### PR TITLE
Add query to load application information from probe info service

### PR DESCRIPTION
**Pull request by scholtzan**
It would be useful to have the information of existing applications provided by the probe info service in BigQuery. This query pulls in the information from the API and dumps it into a table

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1902)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/applications_v1/query.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        bigquery.SchemaField("app_description", "STRING"),
+        bigquery.SchemaField("app_id", "STRING"),
+        bigquery.SchemaField("app_name", "STRING"),
+        bigquery.SchemaField("bq_dataset_family", "STRING"),
+        bigquery.SchemaField("canonical_app_name", "STRING"),
+        bigquery.SchemaField("document_namespace", "STRING"),
+        bigquery.SchemaField("notification_emails", "STRING", mode="REPEATED"),
+        bigquery.SchemaField("url", "STRING"),
+        bigquery.SchemaField("v1_name", "STRING"),
+        bigquery.SchemaField("is_glean", "BOOL"),
```
Comment (akkomar) [inline sql/moz-fx-data-shared-prod/telemetry_derived/applications_v1/query.py:48]: This field is missing in the schema.yaml.

File: sql/moz-fx-data-shared-prod/telemetry/applications/view.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.applications`
```
Comment (akkomar) [inline sql/moz-fx-data-shared-prod/telemetry/applications/view.sql:2]: Nit/observation: `telemetry` dataset becomes a bit overloaded, historically it was for legacy telemetry.

**Review by akkomar — CHANGES_REQUESTED** [review]
`generate-dags` test is failing.

File: sql/moz-fx-data-shared-prod/telemetry/applications/view.sql (hunk 2 above)
Comment (scholtzan) [reply sql/moz-fx-data-shared-prod/telemetry/applications/view.sql:2]: Yeah, I noticed that too... It's basically become the default dataset to use when there is no better dataset. There might be some opportunity to break things up into smaller datasets

**Review by akkomar — APPROVED** [review]

**Pull request merged**